src/models/Task.ts

```diff
@@ -1,7 +1,8 @@
-import { Schema, model } from 'mongoose';
+import { ObjectId, Schema, model } from 'mongoose';
 
 // 1. Create an interface representing a document in MongoDB.
 export interface Task {
+  _id?: ObjectId;
```

Comment:
I don't think _id is an optional field as it's automatically generated by mongodb